modules/example-apache-httpd-configuration.adoc

```diff
@@ -55,6 +57,35 @@ echo "Content-Type: application/json"
 echo ""
 echo '{"error": "Login failure"}'
 exit 0
+
+htpasswd -bc passwords basicauth-xxia1 redhat
+htpasswd -b passwords basicauth-xxia2 redhat
+
+oc create secret generic auth --from-file=passwords
+oc create cm cgi-bin --from-file=login.cgi --from-file=fail.cgi
+oc create cm login.conf --from-file=login.conf
+oc set volumes deployment/$APP --add -t secret --secret-name auth -m /etc/httpd/conf/auth
+oc set volumes deployment/$APP --add -t configmap --configmap-name cgi-bin -m /var/www/cgi-bin --default-mode 0777
+oc set volumes deployment/$APP --add -t secret --secret-name httpd-cert -m /etc/httpd/conf/tls/basic-auth
```

Comment:
Not sure if it is easy for you to launch a cluster to copy and paste the commands to see. If easy, could you try as per your PR's steps? You should pass successfully. But your current PR will let you fail, because, e.g., your PR misses httpd-cert preparation step `oc annotate service $SVC service.beta.openshift.io/serving-cert-secret-name=httpd-cert`.
So I suggest you try your PR's steps, if not pass, re-refer to bug 1955363#c11 to see if missing needed steps. This will make review more efficient without many rounds of reviews.